src/core/core.datasetController.js

```diff
@@ -270,6 +270,7 @@ helpers.extend(DatasetController.prototype, {
 		me.index = datasetIndex;
 		me._cachedMeta = meta = me.getMeta();
 		me._type = meta.type;
+		me._configure();
```

Comment:
I think _configure is called 3 times now, could one of those other two be removed now? The one in `Scale.update` I think.

Reply:
Yes, it was. I've removed one of them